src/main/java/ball/characteristics/Ball.java

```diff
@@ -0,0 +1,27 @@
+package ball.characteristics;
+
+public class Ball {
+    private int number;
+    private String color;
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    @Override
```

Comment:
The toString() method should return a string that represents the entire Ball object, not just the color. Consider including both the number and the color in the returned string.